src/main/java/core/basesyntax/strategy/impl/ReturnOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.impl;
+
+import static javax.swing.UIManager.get;
+import static javax.swing.UIManager.put;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        int previousQuantity = (int) get(transaction.getFruit());
+        put(transaction.getFruit(), previousQuantity + transaction.getQuantity());

```

Comment:
what if the quantity is less than zero? Please fix the same problem in all places. Also, why do you use javax.swing.UIManager.get and javax.swing.UIManager.put. Do you really know what are these methods responsible for? i think you are using them for the wrong purpose. Please test your solution once again and use Storage directly in your handlers in order to get or put values there